views/index.ejs

```diff
@@ -259,7 +259,9 @@
 		</div>
 		
 		<div v-if="!list" class="d-flex mt-3 flex justify-content-center flex-wrap" id="pricelist">
-			<div class="item item-grid m-1" v-for="item in pricelistSorted" :key="item.sku" :style="{ backgroundImage: `url( ${item.style.image_small} ), url( ${item.style.effect} )`, backgroundColor: item.style.quality_color, borderStyle: item.style.craftable? false : 'dashed', opacity: item.enabled?1:0.5, borderColor: item.style.border_color}" @click="itemClick(item, $event)">
+			<div class="item item-grid m-1" v-for="item in pricelistSorted" :key="item.sku" 
+				:style="{ backgroundImage: `url( ${item.style.image_small} ), url( ${item.style.effect} )`, backgroundColor: item.style.quality_color, borderStyle: item.style.craftable? false : 'dashed', opacity: item.enabled?1:0.5, borderColor: item.style.border_color}"
+				@click="itemClick(item, $event)" @mouseenter="mouseEntered(item, $event)">
```

Comment:
either make the decision, in general HTML, to put attributes on all new lines, or put them all on the same line. this will avoid confusion.